marketplace-tests/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>syndesis-qe-parent</artifactId>
+		<groupId>io.syndesis.qe</groupId>
+		<version>0.1-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>marketplace-tests</artifactId>
```

Comment:
it is needed to have it as separate module instead of adding it to `ui`?

Reply:
I understand the intention, but the question is if it is worth for small amount of tests?